src/main/java/Backend.java

```diff
@@ -248,35 +206,35 @@ private void test(String mailbox, String target) {
                 OtpErlangTuple msg_invoke = makeErlangMessage(0, "invoke", "increment", 2);
                 System.out.println("sending invoke message");
                 myOtpMbox.send(mailbox, target, msg_invoke);
-                System.out.println("Recieving message");
+                System.out.println("Receiving message");
                 OtpErlangBinary ErlObjectId2 = (OtpErlangBinary) myOtpMbox.receive();
                 decodeErlangMessage(0, ErlObjectId2);
 
                 OtpErlangTuple msg_invoke2 = makeErlangMessage(0, "invoke", "decrement", 1);
                 System.out.println("sending invoke message");
                 myOtpMbox.send(mailbox, target, msg_invoke2);
-                System.out.println("Recieving message");
+                System.out.println("Receiving message");
                 OtpErlangBinary ErlObjectId3 = (OtpErlangBinary) myOtpMbox.receive();
                 decodeErlangMessage(0, ErlObjectId3);
 
                 OtpErlangTuple msg_invoke3 = makeErlangMessage(0, "read");
                 System.out.println("sending invoke message");
                 myOtpMbox.send(mailbox, target, msg_invoke3);
-                System.out.println("Recieving message");
+                System.out.println("Receiving message");
                 OtpErlangBinary ErlObjectId4 = (OtpErlangBinary) myOtpMbox.receive();
                 decodeErlangMessage(ErlObjectId4);
             } catch (Exception e) {
                 e.printStackTrace();
                 System.exit(1);
             }
         }
-    }
+    } */
 
     // Function to help get the erlang binary representation of the blob for testing
     private void sendbin(OtpErlangBinary bin) {
```

Comment:
if you do the "make a testing subclass" thing then you'd need to make this protected.

Reply:
I don't think this is necessary as I just moved it to BackendWithTesting.java